cbsurge/stats/GDALRasterSource.py

```diff
@@ -49,23 +49,25 @@ def delete_file(self):
 
 
     def reproject(self,
-                  target_srs,
+                  target_crs,
                   resample_alg=gdal.GRA_Bilinear,
                   data_format="GTiff"):
         """
         Reproject the raster file to a specified coordinate reference system.
 
         Parameters:
-            target_srs (str): EPSG code or PROJ.4 string for the target projection.
+            target_crs (str): EPSG code or PROJ.4 string for the target projection.
```

Comment:
@Thuhaa you should change here as well (I did not tell you this place because it is probably easy for you to find this from other comment...)